views/trust-information.html

```diff
@@ -67,7 +47,8 @@ <h2 class="govuk-heading-m">Enter trust information from the Excel document</h2>
               text: "If you've already completed the trust Excel document, follow the instructions in the document to paste the data into the box:",
               classes: "govuk-body",
               isPageHeading: false
-            }
+            },
+            value: trusts_input
```

Comment:
It seems the `trusts_input` is not passed in the trust page in the get controller.

Reply:
This isn't needed for the `get` as there isn't data to be displayed.

Reply:
I think the user wants to visualise what has been added after saving it

Reply:
I've tested this locally and that's not the case. The agreed user journey is that no saved data will be displayed until the Check Your Answers page.
This trusts_input is to pre-populate the field with the submitted data, only when validation has failed.

Reply:
just to confirm, the trust input box should be empty if the user navigates back to the page after entering valid json?  but we would need to show the json if there is an error.

Reply:
Yes, that is correct

Reply:
Something like `trusts_input: appData[TrustKey]?.toString() ` probably will do, but for MVP can be done in another task